compilers.md

```diff
@@ -164,6 +164,8 @@ Microsoft .NET. FTN95 ships with the world's best runtime checking and a great
 range of supporting software. All standard and many vendor-specific legacy
 language features are supported, so that Fortran projects may be any combination
 of Fortran 77, Fortran 90 and Fortran 95.
+Some features of Fortran 2003 and 2008 have been [added](https://www.silverfrost.com/19/ftn95/support/ftn95_revision_history.aspx).
+Silverfrost Fortran runs on both 32 and 64-bit Windows. There is a free personal edition.
```

Comment:
To be consistent with other compiler descriptions. x86 support is implied in a way.
```suggestion
Silverfrost Fortran runs on Windows / x86_64. There is a free personal edition.
```